mongoclient.js

```diff
@@ -0,0 +1,8 @@
+(function() {
```

Comment:
I'm not sure you need this file. I see that your are requiring mongodb.MongoClient *and* this file. I there a difference between them?